README.md

```diff
@@ -56,7 +56,8 @@ git push -u origin new_branch_name
 
 Then in your forked repo, you can send a Pull Request. If this is your first
 time contributing to a Google Open Source project, you may need to follow the
-steps in [contributing.md](contributing.md).
+steps in [contributing.md](contributing.md). Be sure to follow STYLE_GUIDE.md
```

Comment:
For consistency, do [STYLE_GUIDE.md](STYLE_GUIDE.md) like for contributing.md?

Reply:
I think Tiffany was suggesting adding a link to the style guide here.